### PR TITLE
handle strong run_exports

### PR DESCRIPTION
**Pull request by CJ-Wright**
This way we combine run and build the rebuild picking if strong run exports

CC @isuruf

### CODE REVIEW

File: conda_forge_tick/auto_tick.xsh [hunk 1]
```diff
@@ -475,7 +475,12 @@ def add_rebuild_migration_yaml(migrators, gx, package_names, yaml_contents,
     for node, node_attrs in gx.nodes.items():
         attrs = node_attrs['payload']
         meta_yaml = attrs.get("meta_yaml", {}) or {}
-        bh = get_requirements(meta_yaml, run=False, build=False, host=True) or get_requirements(meta_yaml, build=Truerun=False, host=False)
+        if ('strong' in meta_yaml.get('build', {}) or
```
Comment (isuruf) [inline conda_forge_tick/auto_tick.xsh]: This is looking at strong run_exports of the downstream package? You need to check in the upstream package

Comment (CJ-Wright) [reply conda_forge_tick/auto_tick.xsh]: I'm confused.

Comment (CJ-Wright) [reply conda_forge_tick/auto_tick.xsh]: Actually can we leave this off for now? It seems that strong exports are limited to the compilers and most likely aren't in the python graph

Comment (isuruf) [reply conda_forge_tick/auto_tick.xsh]: Sure, but can you open an issue so that it is tracked?

Comment (CJ-Wright) [reply conda_forge_tick/auto_tick.xsh]: https://github.com/regro/cf-scripts/issues/677

**Pull request merged**